simtools/applications/make_regular_arrays.py

```diff
@@ -12,6 +12,10 @@
 
     Command line arguments
     ----------------------
+    site (str, required)
+        North or South.
```

Comment:
It is usually clear when we talk about site we mean the CTA site but in other places in the code we started specifying the observatory as well. Not that I prefer to specify the observatory (I don't) but I want to call your attention to this.

Reply:
Good points, I've improved the docstring.